pcsx2/DEV9/net.cpp

```diff
@@ -49,19 +53,22 @@ void InitNet(NetAdapter* ad)
 	nif = ad;
 	RxRunning = true;
 
+	rx_thread = std::thread(NetRxThread);
+
+#ifdef _WIN32
+	SetThreadPriority(rx_thread.native_handle(), THREAD_PRIORITY_HIGHEST);
+#elif defined(__linux__)
```

Comment:
Not necessarily linux? I assume macos handles the POSIX API just fine

Reply:
Isn't #elif defined(__unix__) more appropiate?

Reply:
Would `#elif defined(__unix__) || defined(__APPLE__)` cover everything?

Reply:
defined unix is used for linux and apple in the code recently, only config.inl uses that specific line so i guess it would work too.

Reply:
yeah, this should cover everything

Reply:
> defined unix is used for linux and apple in the code recently, only config.inl uses that specific line so i guess it would work too.

A Google srarch seemed to impy that __unix__ wasn't defined when compiling for mac. Hence the extra check. Are you saying it actually is?

Reply:
`__UNIX__` is defined on macOS by wx, I don't think we use it anywhere though
`__unix__` is not defined on macOS
`__POSIX__` is defined on macOS by `common/include/Pcsx2Defs.h`

If the functions you're using are defined by posix (I think they are in this case), you should probably use `__POSIX__`
If not, but they happen to exist on both Linux and macOS, use `defined(__linux__) || defined(__APPLE__)`

Reply:
Ah, so ``__POSIX__`` is what I should use in this case then, given it's just posix threads?
``common/include/Pcsx2Defs.h`` is included in `PrecompiledHeader.h` so I shouldn't need to include anything extra to get it